src/CONFIG.js

```diff
@@ -54,6 +54,7 @@ export default {
         PARTNER_PASSWORD: lodashGet(Config, 'EXPENSIFY_PARTNER_PASSWORD', 'e21965746fd75f82bb66'),
         EXPENSIFY_CASH_REFERER: 'ecash',
         CONCIERGE_URL: conciergeUrl,
+        CONCIERGE_PATHNAME: 'concierge/',
```

Comment:
Move it before line 56. Rename it to `CONCIERGE_URL_PATHNAME` then change the `CONCEIRGE_URL` to `${expensifyURL}concierge/` from line 26.